coverage/src/test/java/com/github/lizardev/xquery/saxon/coverage/MultipleTraceExtensionsTest.java

```diff
@@ -17,7 +17,7 @@ public void shouldCollectCoverageWhenMultipleTraceExtensionsAreProvided() {
         ModuleReport moduleReportForTwoTraceExtension = coverageServiceForMultipleTraceExtensionsCases.getReport().getModuleReport(TOUR_MODULE);
         assertThat(moduleReportForOneTraceExtension.getLineCoverage())
                 .isEqualTo(moduleReportForTwoTraceExtension.getLineCoverage());
-        assertThat(traceExtensionStubForMultipleTraceExtensionsCases.getTraceCodeInjector().numberOfInjectedInstructions()).isEqualTo(258);
-        assertThat(traceExtensionStubForMultipleTraceExtensionsCases.getTraceListener().numberOfEnteredInstructions()).isEqualTo(252415);
+        assertThat(traceExtensionStubForMultipleTraceExtensionsCases.getTraceCodeInjector().numberOfInjectedInstructions()).isEqualTo(260);
+        assertThat(traceExtensionStubForMultipleTraceExtensionsCases.getTraceListener().numberOfEnteredInstructions()).isEqualTo(117827);
```

Comment:
Curious why number of entered instructions dropped.
I have compared coverage of tour module on both versions and looks the same so should be fine.